accesshandler/pkg/providers/aws/flask/flask.go

```diff
@@ -0,0 +1,113 @@
+package flask
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/identitystore"
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/common-fate/granted-approvals/accesshandler/pkg/types"
+	"github.com/common-fate/granted-approvals/pkg/cfaws"
+	"github.com/common-fate/granted-approvals/pkg/gconfig"
+	"github.com/invopop/jsonschema"
+)
+
+type Provider struct {
+	ecsClient        *ecs.Client
+	ssoClient        *ssoadmin.Client
+	iamClient        *iam.Client
+	ssmClient        *ssm.Client
+	cloudtrailClient *cloudtrail.Client
+	idStoreClient    *identitystore.Client
+	orgClient        *organizations.Client
+	awsAccountID     string
+
+	// configured by gconfig
```

Comment:
Add an extra space here to avoid Go associating this comment with the specific value below it (`ecsClusterARN`)
```suggestion
// the below fields are configured by gconfig

```